docs/CMakeLists.txt

```diff
@@ -202,6 +202,11 @@ function(build_docs)
                       COMMENT "Generating OpenVINO documentation"
                       VERBATIM)
 
+    add_custom_target(docs_check
+                      COMMAND ${Python3_EXECUTABLE} ${LINKCHECKER_EXECUTABLE} ${DOCS_BUILD_DIR}/html/ -f ${DOXYGEN_DIR}/linkchecker_filter.yaml --no_recursive -l ${DOCS_BUILD_DIR}
```

Comment:
`LINKCHECKER_EXECUTABLE` is empty since it's not defined in the project. Should we add `find_host_program`?

Reply:
paths like `${DOCS_BUILD_DIR}/html/` must be quoted with `"`